troubleshooting.md

```diff
@@ -12,8 +12,8 @@ it is likely that the `/etc/containers/registries.conf` file is either not insta
 misconfigured.
 
 #### Symptom
-```
-sudo buildah bud -f Dockerfile 
+```console
+# buildah bud -f Dockerfile 
```

Comment:
Is dropping the `sudo` here intentional?

Reply:
yes as the console lighting didn't kick in with it as it was, but I didn't try it with a $ in front, let me try that.